### ISSUE TITLE
[STU-177] When using multiple element types, queue jobs give an error preventing indexing

### ISSUE THREAD

**Issue by a-am** [studioespresso/craft-scout#347]
Craft: 5.5.5
Scout: 5.0.2

I am using multiple element types and when I save an _Entry_ or _Solspace Event Element_ and the queue job tries to run the reindex I get the below error.
```php
$indices[] = \rias\scout\ScoutIndex::create(getenv('ALGOLIA_ENV') . '_search')
  ->getElements(function () {
    return [
      Entry::find()->section(['pages','team','jobs']),
      Event::find()->setRangeStart('now'),
   ];
})->transformer(function (\craft\base\Element $element) {
```
Error:
```php
rias\scout\jobs\MakeSearchable::rias\scout\jobs\{closure}(): 
Argument #1 ($criteria) must be of type rias\scout\jobs\ElementQuery, craft\elements\db\EntryQuery given, 
called in /vendor/illuminate/collections/Arr.php on line 203
```

<sub>[STU-177](https://linear.app/studio-espresso/issue/STU-177/when-using-multiple-element-types-queue-jobs-give-an-error-preventing)</sub>

**Comment by janhenckens** [comment]
Thanks for reporting @a-am, will try to reproduce asap

**Comment by janhenckens** [comment]
Fixed in 5.0.3 - could you update and give it another try @a-am? Thanks!
